google/cloud/bigtable/options.h

```diff
@@ -145,17 +151,29 @@ using DataLimitedErrorCountRetryPolicy =
     ::google::cloud::internal::LimitedErrorCountRetryPolicy<
         bigtable::internal::SafeGrpcRetry>;
 
-/// Option to configure the retry policy used by `Table`.
+/**
+ * Option to configure the retry policy used by `Table`.
+ *
+ * @ingroup bigtable-options
+ */
 struct DataRetryPolicyOption {
```

Comment:
every generated service has an equivalent of this option too. Will they all get these `*-options.dox`?

Reply:
I would like to, yes, but I did the hand-crafted libraries first.